### PR TITLE
Tanking my way downtown, rolling fast, Brussel's past

### PR DESCRIPTION
**Pull request by SwedishNorwegian**
- Added the following tank modules: Optical Rangefinder, Advanced Rangefinder, Coaxial Support Cannon, Hull Mounted LMG, Electro-Mechanical Fire Control, Supercharger, Improved Optics
- Made the following changes to existing modules:
- Additional Machineguns:
`Soft Attack = 1 > 2
Build Cost IC = 0.5 > 0.75
Air Attack = 0 > 0.3
Limit = 0 > 1 `
- Easy Maintenance
`Reliability = 10% > 8%`
- Auto Loader
`Build Cost IC = 2.5 > 3
Breakthrough = 4 > 3.5
Breakthrough (mult) = 0 > 5%
Soft Attack (mult) = 0 > 5%
Hard Attack (mult) = 0 > 5%
Reliability = 0 > -2%`
- Dozer Blade
`Entrenchment = 1 > 0.5`
- Armor Skirts
`Armor Value = 3 > 6
Breakthrough = 3 > 0
Defense = 0 > 2
Build Cost IC = 0.5 > 2`
- Squeezebore Adaptor
`Piercing = +10% > +20%
Hard Attack (mult) = 0 > +5%`
- Wet Ammo Storage
`Reliability = 15% > 12.5%`
- Extra Ammo Storage
`Defense = 4 > 1
Armor Value = 0 > -15%`
- Smoke Launchers
`Defense = 2 > 2.25
Breakthrough = 1 > 1.25
Build Cost IC = 0.5 > 0.25`
- Stabilizer
`Breakthrough = 5 > 0
Hard Attack (mult) = 0 > 7.5%
Soft Attack (mult) = 0 > 10%
Build Cost IC = 2 > 3`
- Radio 1
`Defense = 20% > 10%
Breakthrough = 15% > 10%`
- Radio 2
`Breakthrough = 0 > 4
Breakthrough (mult) = 25% > 20%
Defense = 0 > 3
Defense (mult) = 40% > 20%`
- Radio 3
`Breakthrough = 0 > 5
Breakthrough (mult) = 40% > 30%
Defense = 0 > 4
Defense (mult) = 60% > 30%`
- Engine ( Diesel )
`Maximum Speed = 0 > 0.5
Build Cost IC = 1 > 1.5`
- Engine ( Petrol Electric )
`Fuel Consumption = 2 > 1
Reliability = -50% > -35%
Maximum Speed = 0 > 1
Maximum Speed (mult) = 0 > -7.5%`
- Armor ( Riveted )
`
Hardness (mult) = 0 > -3%
Build Cost IC (mult) = -2% > -1%`
- Armor ( Cast )
`Defense = 3 > 4
Breakthrough = 3 > 4
Hardness = 0.02 > 0.04
Armor Value (mult) = 0 > +15%
Build Cost IC = +20% > 0`
- Armor ( Welded )
`Hardness = 0 > 0.04
Armor Value (mult) = +30% > +15%`

The New Modules have the following stats:
- Supercharger
`Maximum Speed: +1
Maximum Speed (mult): +15%
Fuel Consumption: +1
Limit: 1
Build Cost IC: +2.5
`
- Hull Mounted LMG
`Breakthrough: 2
Soft Attack: 3
Hardness: -0.02
Limit: 1
Build Cost IC: 2`
- Optical Rangefinder
`Breakthrough: 2
Recon: 0.25
Soft Attack (mult): 2.5%
Limit: 1 (Rangefinder)
Build Cost IC: 0.5`
- Advanced Rangefinder
`Breakthrough: 3
Recon: 0.75
Soft Attack (mult): +2.5%
Hard Attack (mult): +2%
Limit: 1 (Rangefinder)
Build Cost IC:0.75`
- Improved Optics
`Soft Attack: +1
Recon Factor: +0.6
Limit: 1`
- Electro-Mechanical Fire Control
`Hard Attack: +2
Piercing: +1
Reliability: -0.02
Limit: 1`
- Coaxial Support Cannon
`Soft Attack: +7
Breakthrough: +2
Reliability: -0.05
Build Cost IC: 2.5`

The Define `ARMOR_VS_AVERAGE` has been changed
`ARMOR_VS_AVERAGE: 0.4, > 0.2,`

This change will reduce the effectiveness of Space Marine divisions quite heavily.

Light Tanks have gotten their terrain attack modifiers buffed slightly.

`Forest: -20% > -17.5%`
`Mountain: -10% > -7.5%`
`Marsh: -10% > -7.5%`
`Plains: 0 > +2.5%`
`Desert: 0 > +2.5%`
`Urban: -40% > -35%`
`Jungle: -30% > -25%`
`River: -20% > -15%`

Armored Cars ( AC ) have been buffed, basically across the board:

`Armored Car (Unit): Suppression = 2.5 > 4`
1914 AC
`Recon = 0 > 1
Soft Attack = 6 > 9
Hard Attack = 2 > 3
Piercing = 6 > 9
`
1934 AC
`Recon = 0 > 2
Soft Attack = 6 > 11
Hard Attack = 2 > 5
Piercing = 6 > 9`
1941 AC
`Recon = 0 > 2
Soft Attack = 16 > 24
Hard Attack = 8 > 12
Piercing = 36 > 48`
1941 AC-AT
`Recon = 0 > 1
Soft Attack = 12 > 24
Hard Attack = 14 > 28
Piercing = 62 > 84`

### CODE REVIEW

**Comment by marijn2112** [comment]
Why remove s-heavy tank battalion?

**Review by marijn211 — COMMENTED** [review]
Seems ready, error log clean, doesnt seem to affect historical, description sounds good
All I wanna verify is the diff, but I gotta wait until the bugfixes can be merged to do that

**Review by marijn211 — APPROVED** [review]

**Pull request merged**
